src/FsSpreadsheet/FsWorkbook.fs

```diff
@@ -39,15 +41,15 @@ type FsWorkbook() =
     /// <summary>
     /// Adds an empty FsWorksheet with given name to the FsWorkbook.
```

Comment:
Summary should be: "Creates an empty FsWorksheet with given name and adds it to the FsWorkbook."